vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -2374,6 +2385,29 @@ class TimelineFragment @Inject constructor(
         }
     }
 
+    /**
```

Comment:
we could avoid this comment by including the context in the function name `displayThreadsBetaOptInDialog`

Reply:
The truth is thats a better name !